code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -20,15 +20,15 @@
 	firemodes = list(
 		FULL_AUTO_400,
 		SEMI_AUTO_NODELAY,
-		list(mode_name="3-round bursts", burst=3, fire_delay=null,     dispersion=list(0.0, 0.6, 1.0), icon="burst"),
-		list(mode_name="short bursts",   burst=5, fire_delay=null,     dispersion=list(0.6, 1.0, 1.0, 1.0, 1.2), icon="burst"),
+		list(mode_name="3-round bursts", burst=3, fire_delay=null,     dispersion=null, icon="burst"),
+		list(mode_name="short bursts",   burst=5, fire_delay=null,     dispersion=null, icon="burst"),
```

Comment:
What's up with this incomplete dispersion removal? If you wanted to remove all dispersion, remove the variables too. Including the ones in defines like `FULL_AUTO_400`.

Reply:
The dispersion removal is so that you can still have a 3 rnd burst, but dispersion is handled by the native guncode rather than a snowflake dispersion list

Reply:
If `dispersion` var does nothing now, just remove it. Don't leave it nulled out.